utils/spdx/src/test/kotlin/model/SpdxExternalReferenceTest.kt

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (C) 2023 The ORT Project Authors (see <https://github.com/oss-review-toolkit/ort/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.utils.spdx.model
+
+import io.kotest.assertions.json.shouldEqualJson
+import io.kotest.core.spec.style.WordSpec
+import io.kotest.matchers.collections.shouldContainExactly
+
+import org.ossreviewtoolkit.utils.spdx.SpdxModelMapper
+
+class SpdxExternalReferenceTest : WordSpec({
+    "Serializing a categories" should {
```

Comment:
Nit: "Serializing categories" or "Serializing a category". Same for deserializing below.